linux_os/guide/system/permissions/files/file_permissions_var_log_stig/rule.yml

```diff
@@ -0,0 +1,35 @@
+documentation_complete: true
+
+
+title: 'Verify permissions of log files'
+
+description: |-
+    Any operating system providing too much information in error messages
+    risks compromising the data and security of the structure, and content
+    of error messages needs to be carefully considered by the organization.
+
+    Organizations carefully consider the structure/content of error messages.
+    The extent to which information systems are able to identify and handle
+    error conditions is guided by organizational policy and operational
+    requirements. Information that could be exploited by adversaries includes,
+    for example, erroneous logon attempts with passwords entered by mistake
+    as the username, mission/business information that can be derived from
+    (if not stated explicitly by) information recorded, and personal
+    information, such as account numbers, social security numbers, and credit
+    card numbers.
+
```

Comment:
The description doesn't describe what exaclty is checked. I think that the description should contain information that all files in /var/log and subdirectories are expected to have mode 0640 and also I suggest adding the list of the excluded file paths to the description.